### DISCUSSION TITLE
posts/rsocket/2

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#372] [Announcements]
# posts/rsocket/2

背景 上一篇博客《通过自定义注解实现REST风格的RSocket CRUD应用》 (1.md)介绍了在Spring如何开发RSocket应用，由于RSocket是不同于Http的新的通信协议，目前常见的压测工具都没有提供现成性能测试支持。 Jmeter中提供了Java Request压测方式，可以对Java方法进行压测，我们可以在Java方法中实现RS...

http://localhost:8080/posts/rsocket/2.html

**Comment by pc-dong** [comment]
test
